gitconfig

```diff
@@ -0,0 +1,27 @@
+[core]
+  excludesfile = ~/.gitignore
+  autocrlf = input
+[color]
+  ui = true
+[fetch]
+  prune = true
+[pull]
+  rebase = true
+[alias]
+  hist = log --pretty=tformat:'%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%ar)%Creset [%an]' --graph --date=short
+  lg = !git --no-pager log --graph --pretty=tformat:'%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%ar)%Creset [%an]' --abbrev-commit --date=short -20
+  lga = !git --no-pager log --graph --pretty=tformat:'%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%ar)%Creset [%an]' --abbrev-commit --date=short -20 --all
+  aa = add --all
+  ap = add --patch
+  ci = commit
+  co = checkout
+  cob = checkout --track -b
+  st = status --short
+  pr = pull --rebase
+  pom = push origin master
+  up = !git pull --rebase --prune $@ && git submodule update --init --recursive
+  wipe = !git add -A && git commit -qm 'WIPE SAVEPOINT' && git reset HEAD~1 --hard
+  bclean = "!f() { git branch --merged ${1-master} | grep -v " ${1-master}$" | xargs git branch -d; }; f"
+  bdone = "!f() { git checkout ${1-master} && git up && git bclean ${1-master}; }; f"
+[include]
+  path = ~/.gitconfig.local
```

Comment:
@roman-dubrovsky, should I duplicate all git aliases here? I don't think so. What aliases you would like to add to that gitconfig also?